src/main/java/com/alipay/oceanbase/hbase/result/ClientStreamScanner.java

```diff
@@ -73,8 +106,19 @@ public Result next() throws IOException {
                 return null;
             }
 
+
+            byte[][] familyAndQualifier = new byte[2][];
```

Comment:
I guess `byte[][] familyAndQualifier;` is ok

Reply:
no， Two-dimensional array needs to specify the dimension